.clang-tidy

```diff
@@ -4,15 +4,19 @@ Checks: "*,
         -altera-*,
         -android-*,
         -fuchsia-*,
-        -google-*,
+        google-*,
+        -google-runtime-int,
         -llvm*,
         -modernize-use-trailing-return-type,
         -zircon-*,
-        -readability-else-after-return,
-        -readability-static-accessed-through-instance,
-        -readability-avoid-const-params-in-decls,
-        -cppcoreguidelines-non-private-member-variables-in-classes,
-        -misc-non-private-member-variables-in-classes"
+        readability-else-after-return,
+        readability-static-accessed-through-instance,
+        readability-avoid-const-params-in-decls,
+        cppcoreguidelines-non-private-member-variables-in-classes,
+        misc-non-private-member-variables-in-classes,
+        -misc-include-cleaner,
```

Comment:
I'd rather keep the include cleaner. I find it quite useful.

Reply:
done